README.md

```diff
@@ -60,3 +60,10 @@ Output:
 ```
 map[ListBuckets:1]
 ```
+
+## Developing
+
+This repo uses a modified version of aws-sdk-go's codegen to produce the code in the `counter/` directory.
+The imports in the codegen assume that it exists in the `github.com/aws/aws-sdk-go` package.
+Thus, for the codegen to work properly, you must clone this repo into `$GOPATH/github.com/aws/aws-sdk-go`.
```

Comment:
No suggestions on how to fix, but this feels unintuitive to me, and while I don't have a high comfort level with my knowledge of go dependencies, it seems like it could lead to confusion in the future.

Reply:
Yeah it's not great. The way around this would be to carefully rewrite `import` statements in the codegen to reference `github.com/Clever/` instead of `github.com/aws`. I can try this

Reply:
no more cloning into non-clever path 03f7846